azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/CallingFragment.kt

```diff
@@ -275,19 +306,45 @@ internal class CallingFragment :
             context?.applicationContext?.getSystemService(Context.POWER_SERVICE) as PowerManager
         wakeLock =
             powerManager.newWakeLock(PowerManager.PROXIMITY_SCREEN_OFF_WAKE_LOCK, javaClass.name)
-        wakeLock.acquire()
+
         sensorManager.registerListener(
             this,
             sensorManager.getDefaultSensor(Sensor.TYPE_PROXIMITY),
             SensorManager.SENSOR_DELAY_NORMAL
         )
+
+        context?.let { context ->
+            if (isTablet(context)) {
+                val isLandScape =
+                    resources.configuration.orientation == Configuration.ORIENTATION_LANDSCAPE
+                val captionsWrapperLayout =
+                    captionsWrapper.layoutParams as ConstraintLayout.LayoutParams
+                captionsWrapperLayout.matchConstraintPercentWidth =
+                    if (isLandScape) 0.33f else 0.45f
+                captionsWrapper.layoutParams = captionsWrapperLayout
+            }
+
+            viewLifecycleOwner.lifecycleScope.launch {
+                viewModel.isCaptionsVisibleFlow.collect {
+                    val height = if (it) 150 else 0
```

Comment:
Magic #'s to Constants, or preferable, dimens.xml or resource files that can be configuration selected better.